lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -1072,7 +1072,7 @@ end
 ---@param ply userdata
 function RuntimeContextBuilder:withOwner(ply)
 	self.player = assert(ply)
-	self.uid = (self.player.UniqueID and self.player:UniqueID()) or "World"
+	self.uid = (IsValid(self.player) and self.player.UniqueID and self.player:UniqueID()) or "World"
```

Comment:
`ply` (`self.player = assert(ply)`) is nil checked (by aforementioned assert) so you should use `self.player:IsValid()` instead. ((Actually all of these should be changed to `ply` since it's equal to `self.player` but you don't need to change that if you don't want to, slightly out of scope))